server/src/main/java/org/opensearch/common/blobstore/EncryptedBlobStore.java

```diff
@@ -65,6 +66,15 @@ public BlobContainer blobContainer(BlobPath path) {
         return new EncryptedBlobContainer<>(blobContainer, cryptoHandler);
     }
 
+    /**
+     * Reoload blobstore metadata
```

Comment:
Typo on "Reload"

Reply:
```suggestion
     * Reload blobstore metadata
```

Reply:
Do you mean that how we can ensure adding a new impl in base always gets addressed forcibly at compile time? If yes, then we can't unless we change the base abstractions and inherit them instead of composing them and changing base will require changes across all repository plugins.

Reply:
But it won't also make sense to change repository plugins because underlying repository plugins don't care about encryption. So, we are stuck in the middle due to making repository an encrypted type.